### PR TITLE
Fix/fix two player graph class

### PR DESCRIPTION
**Pull request by MuvvalaKaran**

### CODE REVIEW

**Comment by MuvvalaKaran** [comment]
Hey, 

I also fixed a bug in DFA construction and reopened the pull request.

File: src/graph/two_player_graph.py [hunk 1]
```diff
@@ -365,6 +365,7 @@ def __call__(self,
         :return: A concrete/active instance of the TwoPlayerGraph
         """
         self._instance = TwoPlayerGraph(graph_name, config_yaml, save_flag)
+        self._instance._graph = nx.MultiDiGraph(name=graph_name)
```
Comment (watakandai) [inline src/graph/two_player_graph.py:368]: NEVER ACCESS PRIVATE VARIABLES OUTSIDE A CLASS. 
`self._instance._graph = nx.MultiDiGraph(name=graph_name)` is already taken care of inside the constructor of `TwoPlayerGraph`

File: src/graph/product.py [hunk 2]
```diff
@@ -364,8 +373,8 @@ def construct_product_absorbing(self):
                                                                                     _v_a_node,
                                                                                     _v_prod_node,
                                                                                     action=auto_action,
-                                                                                    obs=ap,
-                                                                                    pref=pref)
+                                                                                    obs=ap,)
+                                                                                    # pref=pref)
```
Comment (watakandai) [inline src/graph/product.py:377]: same here.

File: src/graph/dfa.py [hunk 3]
```diff
@@ -55,10 +55,10 @@ def _construct_dfa(self, states, edges):
             self.add_state(_s)
 
             # add init and accepting node attribute
-            if _s == "T0_init":
+            if _s.endswith("init"):
                 self.add_initial_state(_s)
 
-            if _s == "accept_all":
+            if _s.startswith("accept"):
```
Comment (watakandai) [inline src/graph/dfa.py:61]: For now, I think this is fine. In the future, we should keep the constant parameters INIT and ACCEPT in one place and use them to check these lines.

**Review by watakandai — CHANGES_REQUESTED** [review]

**Comment by MuvvalaKaran** [comment]
Updated the PR. Fixed Private variable issue

**Review by watakandai — APPROVED** [review]

**Pull request merged**